website/docs/r/coreos_ignition.html.markdown

```diff
@@ -36,3 +38,35 @@ The following arguments are supported:
   * A rendered Terraform Ignition object
 
 Any change of the above fields will cause a new resource to be created.
+
+## Integration with Ignition provider
+
+The `libvirt_ignition` resource can be integrated with terraform
```

Comment:
We shouldn't mix _Terraform_ and _terraform_. I think we should stick with _Terraform_.